Defra.Cdp.Backend.Api/Endpoints/MigrationEndpoints.cs

```diff
@@ -19,10 +20,17 @@ public static void MapMigrationEndpoints(this IEndpointRouteBuilder app)
     /**
      * List services that have database migrations available
     */
-    static async Task<IResult> ListServicesWithMigrations(IAvailableMigrations availableMigrations, CancellationToken cancellationToken)
+    static async Task<IResult> ListServicesWithMigrations(IAvailableMigrations availableMigrations, ITenantServicesService tenantServices, string[]? teamIds, CancellationToken cancellationToken)
```

Comment:
Does this mean admin users can only see some services? Would it not be better to include teamIds on the response and let the ui manage it, since we dont have any JWT code in PBE?